src/03_output_crime_rate.R

```diff
@@ -0,0 +1,221 @@
+# 
+# http://apps.who.int/classifications/icd10/browse/2010/en 
+# 
+# 10.Juvenile crime rates. See crime rate definitions above. Note that juvenile rates are based on charges as it is only when a charge is laid that the age of the suspect is determined. 
+# I cannot get the Juenile crime and illicite drug death 
+Violations_selected_list = Violations_list %>% 
+  filter(str_detect(str_to_lower(Violations) , pattern = "all criminal code violations \\(excluding traffic\\)|total violent criminal code violations|homicide|attempted murder|assault|breaking|entering|youth criminal justice act|total drug violation" )) %>% 
+  pull(Violations)
+
+
+# str(Violations_selected_list)
+# Brett's idea
+# Total, all Criminal Code violations (excluding traffic) [50]
+# Total violent Criminal Code violations [100]
+# Homicide [110]
+
+crime_GEO_list = connection %>%
+  count(GEO ) %>%
+  collect()
+# # 237 regions: Policing district/zone. id Police Services Respondent Codes: RESP like 59774
+#  [59774] need to parse out and join to TMF RESP
+
+
+
+
+
+# Only look at data after 2000
+bc_crime_stats <- connection %>%
+  filter(
+    # GEO=="British Columbia",
+    # str_starts( GeoUID, "59"),
+    REF_DATE >="2010", # focus on most recent years
+    Violations %in% Violations_selected_list,  #c("Assault, level 1 [1430]"   ,"Assault, level 2, weapon or bodily harm [1420]"   ) ,  #  ,
+    Statistics  %in% c("Rate per 100,000 population", "Percentage change in rate")
+  ) %>%
+  # filter(REF_DATE  > lubridate::today() - lubridate::years(11))%>%
+  cansim::collect_and_normalize() %>%
+  janitor::clean_names()
+
+bc_resp_lookup = bc_crime_stats %>% 
+  count(geo, geo_uid)
+# 237 resps
+
+# policy zone is like: Colwood, British Columbia, Royal Canadian Mounted Police, municipal [59819]
+
+# bc_crime_stats = bc_crime_stats %>%
+#   mutate(
+#     `Period Starting` = lubridate::ym(ref_date),
+#     Source = paste("Statistics Canada. Table", cansim_id, "Crime Rate per 100,000 population")
+#   ) %>%
+#   filter(`Period Starting` > lubridate::today() - lubridate::years(11))# %>%
+# select(`Period Starting`, Geo, Geouid, Violations, VECTOR, Value,Statistics, Source)
+
+# From TMF get the look-up table for RESP
+# in TMF, there are only 193 RESPs, but statscan crime rate data has 237 RESPs. 
+
+
+##########################################################################
+# For CSD or DAs, many CSDs or DAs have to share RESP together, so it is better to calculate the ratios within each RESP and CSDs or DAs could share the ratios. 
+# If one CD or CSD has two or more RESPs, we could average them weighting by the number of the postal code regions within the RESPs. 
+# Econ team provides a lookup table for us to link DA to RESP weighted by postal code or population
+###########################################################################
+
+
+DA_RESP_lookup = read_excel(path = use_network_path("data/crime_rate/Pop by DA and RESP.xlsx"), 
```

Comment:
Similar problem as before. This works:

```DA_RESP_lookup = readxl::read_excel(path = "G:/Operations/Data Science and Analytics/2024 SES Index/data/crime_rate/Pop by DA and RESP.xlsx", sheet = "DA RESP")
```

I'm still not sure what the best way to do this is - .Renviron file?

Reply:
I see. We need to set_network_path first. `config.yml` or .Renviron may work better.

Reply:
Use the safepaths::set_network_path function to set up the network path first, and it should work.